.github/workflows/cancel-old-reviews.yaml

```diff
@@ -0,0 +1,17 @@
+on:
+    schedule:
+        -   cron: '0 2 * * *'
+name: Cancel reviews older than seven days
+jobs:
+    rebuild-master-images:
+        name: Cancel reviews older than seven days
+        runs-on: [ self-hosted, linux, review-stage ]
+        steps:
+            -   name: GIT checkout branch - master
+                uses: actions/checkout@v3
+                with:
+                    ref: 'refs/heads/master'
+            -   name: Cancel reviews older than seven days
+                run: |
+                    chmod +x ./.github/cancel-old-reviews.sh
+                    ./.github/cancel-old-reviews.sh
```

Comment:
you can run the file with `bash ./.github/cancel-old-reviews.sh`. That way it's not necessary to set the executable bit.